medical-portal/src/API/RSBC.DMF.MedicalPortal.API/Controllers/CasesController.cs

```diff
@@ -137,6 +138,18 @@ private string TranslateDmerStatus(string dmerStatus, string loginId)
                     dmerStatus = "Required - Claimed";
                 }
             }
+
+            if(dmerStatus == "Non-Comply")
```

Comment:
I think this will now skip "Actioned Non-comply" value. I should have used enums here

Reply:
or is that intentional? I remember that actioned non-comply means we cannot action on the DMER but non-comply means they cannot claim but they can submit (or something like that)

Reply:
This was intentional i have change that Actioned Non Comply at the CMS layer so that when they changed the status it dynamics it will flow through. At the portal level for Non Comply will checked whether that has logged in user or not?